packages/api/pages/api/languages/[code]/verses/[verseId]/footnotes.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  GetVerseFootnotesResponseBody,
+  Footnote,
+} from '@translation/api-types';
+import createRoute from '../../../../../../shared/Route';
+import { client } from '../../../../../../shared/db';
+
+export default createRoute<{ code: string; verseId: string }>()
+  .get<void, GetVerseFootnotesResponseBody>({
```

Comment:
We are making a ton of requests to the server on the translation view, so I think we should consolidate this and the translator's note route. It can be be renamed to end in `/notes` and have to top level keys: `footnotes` and `translatorNotes`

Reply:
Done. @arrocke Do you think we should also consolidate the `/note` and `/footnote` paths?